payment_environment/models/payment_acquirer.py

```diff
@@ -0,0 +1,23 @@
+# Copyright 2021 Camptocamp SA
+# @author Iván Todorovich <ivan.todorovich@gmail.com>
+# License LGPL-3.0 or later (https://www.gnu.org/licenses/lgpl.html)
```

Comment:
AGPL or LGPL? :)

Reply:
after checking it seems there's a bit of confusion on licenses. 

The main module says GPL but refers to AGPL link, the tech mixing [says AGPL](https://github.com/OCA/server-env/blob/13.0/server_environment/models/server_env_mixin.py#L2) but seems wrong to me.

Would you mind opening another PR to fix them so that we can fast track it?

Reply:
PR created here: https://github.com/OCA/server-env/pull/77

As for this, my intention was AGPL, this got carried on from a copy & paste left over.
Fixing it now :sweat_smile: